atomos.maven/src/main/java/org/apache/felix/atomos/maven/NativeImageMojo.java

```diff
@@ -109,78 +86,16 @@ public void execute() throws MojoExecutionException
         try
         {
             Files.createDirectories(outputDirectory.toPath());
-
             Config config = new Config();
-
             config.additionalInitializeAtBuildTime = additionalInitializeAtBuildTime;
-            if (imageName == null || imageName.isEmpty())
```

Comment:
I probably don't need to review this just yet.  It seems the current state is that the Mojos are not hooked into the utilities yet.  The resulting build of atomos.examples.substrate.maven.equinox doesn't produce a native-image yet.